src/inet/TCPEndPointImplSockets.cpp

```diff
@@ -69,9 +69,12 @@ namespace Inet {
 
 CHIP_ERROR TCPEndPointImplSockets::BindImpl(IPAddressType addrType, const IPAddress & addr, uint16_t port, bool reuseAddr)
 {
-    CHIP_ERROR res = GetSocket(addrType);
+    ReturnErrorOnFailure(GetSocket(addrType));
```

Comment:
These changes are touching internals that are sensitive, so split it to #35674

Will resolve this comment and ready to merge only after that PR as a stand-alone merges.

Reply:
I switched to NOLINT for these items, so while 35674 still is open separately, I believe the changes are now sufficiently safe to close this.